mesheryctl/internal/cli/root/perf/view.go

```diff
@@ -0,0 +1,62 @@
+package perf
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var viewCmd = &cobra.Command{
+	Use:   "view",
+	Short: "view perf profile",
+	Long:  `See the configuration of your performance profile`,
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return errors.Wrap(err, "error processing config")
+		}
+		pattern := args[0]
+		var req *http.Request
+		url := mctlCfg.GetBaseMesheryURL() + "/api/user/performance/profiles/" + pattern
+
+		req, err = http.NewRequest("GET", url, nil)
+		if err != nil {
+			return errors.Wrapf(err, utils.PerfError("Failed to invoke performance test"))
+		}
+		err = utils.AddAuthDetails(req, tokenPath)
+		if err != nil {
+			return err
+		}
+		client := &http.Client{}
+		resp, err := client.Do(req)
+		if err != nil {
+			return err
+		}
+		if resp.StatusCode != 200 {
+			// failsafe for the case when a valid uuid v4 is not an id of any pattern (bad api call)
+			return errors.Errorf("Response Status Code %d, possible invalid ID", resp.StatusCode)
```

Comment:
```suggestion
			return errors.Errorf("Performance profile `%s` not found. Please verify profile name and try again. Use `mesheryctl perf list` to see a list of performance profiles.", profileID)
```

Reply:
yes, you are right. thank you i will fix it